demo/README.md

```diff
@@ -0,0 +1,12 @@
+Creating the virtual environment in the root folder:
+
+1. `python -m venv env` Set up the virtual environment
+2. `source env/bin/activate` Activate the environment
+
+Once you have created the virtual environment in the parent folder:
+
+1. Cd to this folder
+2. run `pip install ../.` to install the local (most up to date) version of the integration
+3. Run `flask --app demo run`
+4. Navigate to `http://127.0.0.1:5000`
+5. Login with `demo@phrase.com` / `phrase`
```

Comment:
I had to refresh after logging in to see the ICE, not sure if need to add a point there about that

Reply:
Hmm interesting, I didn't have that. It sounds like bug with the ICE in general, not with the integration

Reply:
I'll make a follow-up ticket for that then

Reply:
Yeah it's strange, I can't reproduce if if I log out and back in. I recall it happening before too... it seems to only happen sometimes, not always. Anyways, created this ticket for follow-up: https://phrase.atlassian.net/browse/TSE-1311